reference-architecture/deploy-arch-ibm-vpc-fully-configurable.md

```diff
@@ -0,0 +1,150 @@
+  - title: "Landing zone for containerized applications with Red Hat OpenShift"
+    url: "https://cloud.ibm.com/docs/deployable-reference-architectures?topic=deployable-reference-architectures-ocp-ra"
+    description: "A deployable architecture that creates secure and compliant Red Hat OpenShift Container Platform workload clusters on a Virtual Private Cloud (VPC) network."
+
+use-case: Cybersecurity
+industry: Banking,FinancialSector
+compliance: FedRAMP
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+# Cloud foundation for VPC - Standard variation
+{: #vpc-fully-configurable}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+{: toc-version="1.0.0"}
+
+The Cloud foundation for VPC deployable architecture sets up a foundational IBM Cloud Virtual Private Cloud (VPC) environment with full configurability and flexibility. This deployable architecture provides complete control over VPC configuration, including subnets, network ACLs, security groups, public gateways, VPN gateways, and VPE gateways. Unlike pre-configured variations, this solution allows you to customize every aspect of your VPC infrastructure to meet specific requirements.
+
+This deployable architecture includes enterprise-grade security, observability, and compliance capabilities through integrated dependencies that are enabled by default: encryption services (Key Protect), storage services (Cloud Object Storage), observability services (Cloud Logs, Cloud Monitoring, Activity Tracker), and compliance services (Security and Compliance Center Workload Protection). These services provide comprehensive security, monitoring, and compliance coverage for your VPC infrastructure.
+
+This architecture lays the groundwork for adding Virtual Server Instances (VSI), Red Hat OpenShift clusters, and other advanced resources. It can be used as a base deployable architecture for many other solutions or as a standalone VPC infrastructure deployment.
+
+## Architecture diagram
+{: #ra-vpc-fully-configurable-architecture-diagram}
+
+![Architecture diagram for the Standard - Integrated setup with configurable services variation of Cloud foundation for VPC](deployable-architecture-vpc.svg "Architecture diagram of VPC deployable architecture"){: caption="Figure 1. Fully configurable variation of Cloud foundation for VPC" caption-side="bottom"}{: external download="deployable-architecture-vpc.svg"}
+
+## Design requirements
+{: #ra-vpc-qs-design-requirements}
+
+![Design requirements for Cloud foundation for VPC](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/main/reference-architectures/heat-map-deploy-arch-slz-vpc-standard.svg "Design requirements"){: caption="Figure 2. Scope of the design requirements" caption-side="bottom"}
```

Comment:
You will need a new heat map - one that has Monitoring, Alerting and Event management highlighted as they are all on by default in this variation